lib/asciidoc-preview-view.coffee

```diff
@@ -162,9 +174,9 @@ class AsciiDocPreviewView extends ScrollView
         # Because jQuery uses CSS syntax for selecting elements, some characters are interpreted as CSS notation.
         # In order to tell jQuery to treat these characters literally rather than as CSS notation, they must be "escaped" by placing two backslashes in front of them.
         if target = $(hrefLink.replace(/(\/|:|\.|\[|\]|,|\)|\()/g, '\\$1'))
-          continue if not target.offset()
+          continue if not target[0]
           # TODO Use tab height variable instead of 43
-          top = target.offset().top - 43
+          top = target[0].offsetTop
```

Comment:
`target.offset().top` depends on the scroll position. During my tests only the first click on a link was working.

`target[0].offsetTop` seems to be working all the time (ie. fixed value).